src/bin/nodes/CommandPing.ts

```diff
@@ -53,7 +53,7 @@ class CommandPing extends CommandPolykey {
           );
         } catch (err) {
           if (err instanceof nodesErrors.ErrorNodeGraphNodeNotFound) {
-            error = new CLIErrors.ErrorNodePingFailed(
+            error = new CLIErrors.ErrorCLINodePingFailed(
```

Comment:
Same situation with `find`. If `ping` doesn't respond, you just wait forever.

It seems like ping would basically do find first and then try to ping the node. So it may wait forever to find/resolve, and then wait forever to ping.

An optional parameter for timing out should be available.

Reply:
Added the same options as CommandFind